packages/govern-subgraph/schema.graphql

```diff
@@ -114,15 +107,6 @@ type Evidence @entity {
   createdAt: BigInt!
 }
 
-type Veto @entity {
```

Comment:
For removing this, on the `data` field of the latest container `history` event that will get recorded I just put the `reason`. That way you should be able to see why this was vetoed.